purchase_request_operating_unit/view/purchase_request_view.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--Copyright 2016-19 Eficent Business and IT Consulting Services S.L.-->
+<!--(http://www.eficent.com)-->
+<!--Copyright 2016-19 Serpent Consulting Services Pvt. Ltd.-->
+<!--(<http://www.serpentcs.com>)-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="view_purchase_request_tree" model="ir.ui.view">
+        <field name="name">purchase.request.tree</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_tree" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field
+                    name="operating_unit_id"
+                    groups="operating_unit.group_multi_operating_unit"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="view_purchase_request_form" model="ir.ui.view">
+        <field name="name">purchase.request.form</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_form" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field
+                    name="operating_unit_id"
+                    groups="operating_unit.group_multi_operating_unit"
+                    domain="[('company_id','=', company_id),
+                     ('user_ids', 'in', requested_by)]"
```

Comment:
```suggestion
                     ]"
```

please, remove this domain because it will filter with 1 ou only.
Example,
Admin has all operating unit but can't see other ou
![Peek 2023-01-31 15-59](https://user-images.githubusercontent.com/20896369/215714672-70585989-5232-41f4-9035-142777b08b5c.gif)